Change wording for rollover and index (#3080)
opensearch-project/documentation-website: _dashboards/im-dashboards/datastream.md

    ![Data Streams]({{site.url}}{{site.baseurl}}/images/admin-ui-index/datastreams3.png)

> 1. The **Inherited settings from template** and **Index alias** sections are read-only. This will inform you of the backing indexes used in the data stream.

Reviewer: Instead of 'this will inform you of' how about, ".... are read-only, and display the backing indexes that are contained in the data stream." ?

Author: Yeah, sounds good, I can make that change.

> 1. The **Inherited settings from template** and **Index alias** sections are read-only. This will inform you of the backing indexes used in the data stream.

Author: ```suggestion
1. The **Inherited settings from template** and **Index alias** sections are read-only, and display the backing indexes that are contained in the data stream.
```

## Rolling over a data stream

> To perform a roll over operation on a data stream, perform the following steps:

Reviewer: Should we explain why someone would want to perform a roll over on a data stream?

Author: This is explained in the ISM/data-stream/API documentation as to the “why” you would do it. As this is a front-end for an existing feature I wouldn’t think so. Also, I have linked to that content in this page.

Reviewer 2: This should be rollover because it's used as an adjective
```suggestion
To perform a rollover operation on a data stream, perform the following steps:
```
